<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QWidget" name="imageWidget" native="true">
      <layout class="QVBoxLayout" name="mplvl"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="controlPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>507</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Gauge Controls</string>
       </attribute>
       <widget class="QPushButton" name="stopStreamButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Stop Stream</string>
        </property>
       </widget>
       <widget class="QPushButton" name="startStreamButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Start Stream</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QSpinBox" name="lengthField">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QLabel" name="lengthLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Display buffer</string>
        </property>
       </widget>
       <widget class="QLabel" name="sampleLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sample delay (ms)</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sampleField">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
       <widget class="QPushButton" name="startTurboButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Start Turbo</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="stopTurboButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>Stop Turbo</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Turbo status:</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusDisplay">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>260</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>None</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="errorDisplay">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>None</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="errorLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Turbo error:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
